.github/workflows/test.yaml

```diff
@@ -0,0 +1,34 @@
+name: Tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: build core in snapcraft
+        run: |
+          export URL=http://cdimage.ubuntu.com/ubuntu-base/releases/16.04/release/ubuntu-base-16.04.6-base-amd64.tar.gz
```

Comment:
instead of using export you could use env
env:
   URL: http://cdimage.ubuntu.com/ubuntu-base/releases/16.04/release/ubuntu-base-16.04.6-base-amd64.tar.gz
   CHROOT: xenial-test-chroot
 .....

Reply:
Thanks, updated!